src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -939,6 +928,47 @@ def body(self):
         finally:
             os.remove(config_file)
 
+class VMSSExtensionInstallTest(VCRTestBase):
```

Comment:
>VCRTestBase [](start = 31, length = 11)

This should be a ResourceGroupTestBase so it doesn't rely on "preexisting" resources. Same with VMExtensionInstallTest above.